ios/CapacitorCordova/CapacitorCordova/Info.plist

```diff
@@ -4,6 +4,8 @@
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>$(DEVELOPMENT_LANGUAGE)</string>
+	<key>CFBundleDisplayName</key>
+	<string>Cordova</string>
```

Comment:
Why not `CapacitorCordova`?

Reply:
Because this matches how it is configured when importing via Cocoapods. The output bundle/module/filename are all `Cordova` despite what the project is named.